src/components/overview/Overview.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import axios from 'axios';
+import ProductImages from './ProductImages/ProductImages.js';
```

Comment:
This file isn't in your PR?

Reply:
Yeah oops. Let me take out those two lines from the code. I wanted it to be a review of each section rather than the whole overview at once.